websites/X/Xuply/dist/metadata.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.0",
+  "author": {
+    "name": "BBQ",
+    "id": "298888568279924746"
+  },
+  "url": "xuply.dev",
+  "description": {
+    "tr": "Xuply discord botları ve daha fazlası için basit bir kod paylaşım platformudur.",
+    "en": "Xuply is a simple code sharing platform for discord bots and more."
+  },
+  "service": "Xuply",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/fMUpAe0.png",
+  "thumbnail": "http://i.imgur.com/a/AiGDoAD.png",
+  "color": "#0bc9e7",
```

Comment:
wrong url it should be: https://i.imgur.com/wTuunx3.png

Reply:
Fixed